src/builtin.c

```diff
@@ -1184,6 +1184,47 @@ static jv f_string_indexes(jq_state *jq, jv a, jv b) {
   return jv_string_indexes(a, b);
 }
 
+static jv f_string_trim(jq_state *jq, jv a) {
```

Comment:
We could jq-code `trim`...

Reply:
Main reason was i couldn't come up with something that i thought would be reasonably performant. Any ideas?   reduce/slice somehow? fq's trim is `def trim: capture("^\\s*(?<str>.*?)\\s*$"; "m").str;` atm but i fear that compiling a regex each time might not be great (maybe a regex cache would be good?)

Maybe time to measure?

Reply:
Good point.